### PR TITLE
Migration ToolKit For Virtualization Common Module 

### PR DESCRIPTION
**Pull request by amastbau**
##### Short description:
Common Module requires by  all MTV Resouce Handlers

### CODE REVIEW

**Review by myakove — APPROVED** [review]

File: ocp_resources/mtv.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        try:
+            for sample in samples:
+                current_conditions = (
+                    sample.items[0].status.get("conditions")
+                    if sample.items and sample.items[0].status
+                    else []
+                )
+                for condition in current_conditions:
+                    last_condition = condition
+                    if (
```
Comment (sarahbx) [inline ocp_resources/mtv.py:91]: Would it be possible to flatten out this logic, save the conditions to boolean vars? It would be much easier to read.

**Review by sarahbx — CHANGES_REQUESTED** [review]

File: ocp_resources/mtv.py (hunk 1 above)
Comment (amastbau) [reply ocp_resources/mtv.py]: wyt now? @sarahbx

**Pull request merged**